src/main/java/com/amazon/opendistroforelasticsearch/jdbc/protocol/http/JdbcCursorQueryRequest.java

```diff
@@ -0,0 +1,59 @@
+package com.amazon.opendistroforelasticsearch.jdbc.protocol.http;
+
+import com.amazon.opendistroforelasticsearch.jdbc.protocol.JdbcQueryParam;;
+import com.amazon.opendistroforelasticsearch.jdbc.protocol.QueryRequest;
+
+
+import java.util.List;
+import java.util.Objects;
+
+public class JdbcCursorQueryRequest implements QueryRequest {
+
+    String cursor;
+    List<JdbcQueryParam> parameters;
```

Comment:
Make `cursor` final and what is the use of `parameters`?

Reply:
- Made `cursor` final. 
- `parameters` are used in `PreparedStatement`.